pkg/devfile/adapters/docker/component/adapter_test.go

```diff
@@ -28,12 +43,13 @@ func TestPush(t *testing.T) {
 			client:        fakeClient,
 			wantErr:       true,
 		},
-		{
-			name:          "Case 2: Valid devfile",
-			componentType: versionsCommon.DevfileComponentTypeDockerimage,
-			client:        fakeClient,
-			wantErr:       false,
-		},
+		// disabling this at the moment, because push requires a valid indexer path for syncing
```

Comment:
Makes sense to me, but we should look at re-enabling this test when we look at improving the Docker client unit tests with the go mock package Jonathan is adding in his PR